src/CMakeLists.txt

```diff
@@ -1,21 +0,0 @@
-add_library(maxminddb
-  maxminddb.c
-  data-pool.c
-)
-
-if(WIN32)
-  target_link_libraries(maxminddb ws2_32)
-endif()
-
-set_target_properties(maxminddb
-  PROPERTIES PUBLIC_HEADER
-  "../include/maxminddb_config.h;../include/maxminddb.h"
-  SOVERSION ${MAXMINDDB_SOVERSION}
```

Comment:
We have lost the soversion when building and installing a shared library.

Before:

```
 Install the project...
-- Install configuration: ""
-- Installing: /tmp/test2/lib/libmaxminddb.so.0.0.7
-- Installing: /tmp/test2/lib/libmaxminddb.so
-- Installing: /tmp/test2/include/maxminddb_config.h
-- Installing: /tmp/test2/include/maxminddb.h
-- Installing: /tmp/test2/bin/mmdblookup
-- Set runtime path of "/tmp/test2/bin/mmdblookup" to ""
```

After:

```
Install the project...
-- Install configuration: ""
-- Installing: /tmp/test2/lib/libmaxminddb.so
-- Up-to-date: /tmp/test2/lib/cmake/maxminddb/maxminddb-config.cmake
-- Installing: /tmp/test2/lib/cmake/maxminddb/maxminddb-config-noconfig.cmake
```

Reply:
This should be good now:

```
build$ make install                     
-- Configuring done
-- Generating done
-- Build files have been written to: /Users/bsergeant/src/foss/github/libmaxminddb/build
[100%] Built target maxminddb
Install the project...
-- Install configuration: ""
-- Up-to-date: /usr/local/lib/libmaxminddb.0.0.7.dylib
-- Up-to-date: /usr/local/lib/libmaxminddb.dylib
-- Installing: /usr/local/include/maxminddb.h
-- Installing: /usr/local/include/maxminddb_config.h
-- Up-to-date: /usr/local/lib/cmake/maxminddb/maxminddb-config.cmake
-- Installing: /usr/local/lib/cmake/maxminddb/maxminddb-config-noconfig.cmake
```